lib/files.test.js

```diff
@@ -1,323 +1,323 @@
+//       expect(fn()).to.equal();
+//     });
 
-    it('handles bad name', function () {
-      const name = 'some name';
+//     it('handles bad name', function () {
+//       const name = 'some name';
 
-      expect(fn(name)).to.equal();
-    });
+//       expect(fn(name)).to.equal();
+//     });
 
-    it('handles bad path', function () {
-      const name = 'some name';
+//     it('handles bad path', function () {
+//       const name = 'some name';
 
-      lib.getComponentPath.returns(null);
+//       lib.getComponentPath.returns(null);
 
-      expect(fn(name)).to.equal(null);
-    });
+//       expect(fn(name)).to.equal(null);
+//     });
 
-    it('handles index.js path', function () {
-      const name = 'some name',
-        path = 'some path',
-        result = [];
+//     it('handles index.js path', function () {
+//       const name = 'some name',
+//         path = 'some path',
+//         result = [];
 
-      lib.getComponentPath.returns(path);
-      req.resolve.withArgs(path + '/package.json').returns(path + '/package.json');
-      req.withArgs(path + '/package.json').returns(result);
+//       lib.getComponentPath.returns(path);
+//       req.resolve.withArgs(path + '/package.json').returns(path + '/package.json');
+//       req.withArgs(path + '/package.json').returns(result);
 
-      expect(fn(name)).to.equal(result);
-    });
-  });
+//       expect(fn(name)).to.equal(result);
+//     });
+//   });
 
-  describe('readFilePromise', function () {
-    const fn = lib[this.title];
+//   describe('readFilePromise', function () {
+//     const fn = lib[this.title];
 
-    it('returns file contents', function () {
-      const result = 'result',
-        name = 'article.css';
+//     it('returns file contents', function () {
+//       const result = 'result',
+//         name = 'article.css';
 
-      sandbox.stub(fs, 'readFile').callsFake(function (path, options, callback) {
-        return callback(null, result);
-      });
+//       sandbox.stub(fs, 'readFile').callsFake(function (path, options, callback) {
+//         return callback(null, result);
+//       });
 
-      fn(name).then(function (fileResult) {
-        expect(fileResult).to.equal(result);
-      });
-    });
+//       fn(name).then(function (fileResult) {
+//         expect(fileResult).to.equal(result);
+//       });
+//     });
 
-    it('throws error', function () {
-      sandbox.stub(fs, 'readFile').callsFake(function (path, x, callback) {
-        return callback(new Error(), '');
-      });
+//     it('throws error', function () {
+//       sandbox.stub(fs, 'readFile').callsFake(function (path, x, callback) {
+//         return callback(new Error(), '');
+//       });
 
-      fn('.').catch(function (result) {
-        expect(result).to.equal('');
-      });
-    });
-  });
-});
+//       fn('.').catch(function (result) {
+//         expect(result).to.equal('');
+//       });
+//     });
+//   });
+// });
```

Comment:
can commented out code be 🔪 'ed? If you need to restore it it will always be in history